QtlDataPass/SecondView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SecondView.swift
+//  QtlDataPass
+//
+//  Created by 柿崎逸 on 2023/04/15.
+//
+
+import SwiftUI
+
+struct SecondView: View {
+    @Binding var textSecond: String
+    var body: some View {
+        Text("ContentView画面で入力した内容が下記に表示されます。")
+        Text(textSecond)    }
```

Comment:
```suggestion
        Text(textSecond)
    }
```

[must] カッコの位置が分かりづらいので改行お願いします

Reply:
コミットハッシュ: c2e095ab8f744f2d57332797fb7c2515a148b25e

変更：TopTextInputViewの15行目 ~ 16行目に改行